server/coin-scraper.js

```diff
@@ -62,22 +62,22 @@ module.exports = class CoinScraper {
     this.mostRecentItems = response.data;
     this.mostRecentItems = Array.isArray(this.mostRecentItems) ? this.mostRecentItems : [];
     this.mostRecentItems.push(
-    {
-      id: 'dummy',
-      symbol: 'hellooo',
-      quote: {
-        USD: {
-          market_cap: 12345.67890
+      {
+        id: 'dummy',
+        symbol: 'hellooo',
+        quote: {
+          USD: {
+            market_cap: 12345.67890
+          }
         }
-      }
-    });
+      });
     const dictionarySize = Object.keys(this.previousItemsMap).length;
     let discoveredAt = null;
     let message = null;
     this.mostRecentItems.map(coin => {
-      if(!this.previousItemsMap[`${coin.id}`] && dictionarySize) {
+      if (!this.previousItemsMap[`${coin.id}`] && dictionarySize) {
         discoveredAt = new moment();
-        message = `new coin, id: ${coin.id}, smbol: ${coin.symbol}, market_cap: ${coin.quote.USD.market_cap}, found at ${discoveredAt.format('HH:mm:ss')} on ${discoveredAt.format('D-MMM-yyyy')}`;
+        message = `new coin, id: ${coin.id}, symbol: ${coin.symbol}, market_cap: ${coin.quote.USD.market_cap} on ${discoveredAt.format('D-MMM-yyyy')}`;
```

Comment:
@Shapokhan i think we can let go of the `on ${discoveredAt.format('D-MMM-yyyy')}` part too, cuz the date and time can be check from discord post easily, check this snapshot
<img width="621" alt="image" src="https://user-images.githubusercontent.com/6904200/116971884-af0eed00-acd3-11eb-883b-8123b9e71a80.png">

Reply:
@Shapokhan i set this one to `.toString()` format (applied to another change above) too.